src/pages/Wallet/WalletOverview/components/WalletInsights/WalletInsights.tsx

```diff
@@ -33,9 +45,53 @@ const WalletInsights = ({ balances }: WalletInsightsProps): JSX.Element => {
       new Big(0)
     );
 
+  const totalLendPositions =
+    lendPositions &&
+    Object.values(lendPositions).reduce(
+      (total, balance) =>
+        total.add(
+          convertMonetaryAmountToValueInUSD(
+            balance.amount,
+            getTokenPrice(prices, balance.amount.currency.ticker)?.usd
+          ) || 0
+        ),
+      new Big(0)
+    );
+
+  const totalBorrowPositions =
+    borrowPositions &&
+    Object.values(borrowPositions).reduce(
+      (total, balance) =>
+        total.add(
+          convertMonetaryAmountToValueInUSD(
+            balance.amount,
+            getTokenPrice(prices, balance.amount.currency.ticker)?.usd
+          ) || 0
+        ),
+      new Big(0)
+    );
```

Comment:
I think these values are already computed in `useGetAccountLendingStatistics` hook, under `supplyAmountUSD` and `borrowAmountUSD` fields, so we don't have to recompute them.

Reply:
Thanks—will use those instead